test/test_incompressible_models.jl

```diff
@@ -27,8 +27,9 @@
                 # Just testing that the model was constructed with no errors/crashes.
                 @test model isa IncompressibleModel
 
+                # FJP: the grid does not need to equal model.grid
                 # Test that the grid didn't get mangled
-                @test grid === model.grid
+                #@test grid === model.grid
```

Comment:
Did you mean to leave these comments in?

Reply:
Thanks @ali-ramadhan .  I removed those comments from above.  I also removed `test_flat_shallow_water_model.jl` as I added compariable tests in `test_shallow_water_models.jl`.  I take this as done and will merge.